pyomo/contrib/mindtpy/iterate.py

```diff
@@ -475,8 +457,8 @@ def fix_dual_bound(solve_data, config, last_iter_cuts):
         else:
             update_suboptimal_dual_bound(solve_data, main_mip_results)
             config.logger.info(
-                'Fixed bound values: LB: {}  UB: {}'.
-                format(solve_data.LB, solve_data.UB))
+                'Fixed bound values: Primal Bound: {}  Dual Bound: {}'.
+                format(solve_data.primal_bound, solve_data.dual_bound))
         # Check bound convergence
-        if solve_data.LB + config.bound_tolerance >= solve_data.UB:
+        if abs(solve_data.primal_bound - solve_data.dual_bound) <= config.bound_tolerance:
```

Comment:
The name of `bound_tolerance` seems to be incomplete. I would rename this as `absolute_bound_tolerance`